src/Microsoft.CodeAnalysis.Workspaces.Dnx/ProjectModel.cs

```diff
@@ -134,17 +135,12 @@ private static DependencyInformation ResolveDependencyInfo(DnxProject project, F
                     }
                 }
 
-                var exportWithoutProjects = ProjectExportProviderHelper.GetExportsRecursive(
-                     cache,
-                     applicationHostContext.LibraryManager,
-                     applicationHostContext.LibraryExportProvider,
-                     new LibraryKey
-                     {
-                         Configuration = "Debug",
-                         TargetFramework = frameworkName,
-                         Name = project.Name
-                     },
-                     library => library.Type != "Project");
+                var libraryExporter = new LibraryExporter(
+                    applicationHostContext.LibraryManager,
+                    null,
```

Comment:
``CompilationEngine``. It's used (inside of ``LibraryExporter``) only for exporting libraries with ``Project`` type and call to ``GetNonProjectExports`` doesn't need it.

Actually in the previous version it worked in the same way but export related classes has been refactored in the DNX betta8. In the previous version ``null`` has been passed as ``IServiceProvider `` to the ``ApplicationHostContext`` and it's used for ``ProjectLibraryExportProvider`` creation inside ``ApplicationHostContext``.